src/types/order.d.ts

```diff
@@ -25,10 +25,11 @@ export interface OrderAddressBase {
   postcode: string
   county: string
   country: string
+  phone_number: string
```

Comment:
I added `phone_number` to OrderAddressBase because it's used for Billing and Shipping addresses in CM. But I also didn't find in docs `phone_number` for Billing Address. So, I'll remove it in CM.
<img width="1002" alt="Screenshot 2022-08-11 at 11 16 27" src="https://user-images.githubusercontent.com/34708483/184092573-6141e351-3994-49bf-8cc5-7d939e0353cf.png">